schema.json

```diff
@@ -117,7 +119,7 @@
             "additionalProperties": {
                 "anyOf": [
                     { "$ref": "#" },
-                    { "$ref": "#/definitions/stringArray" }
+                    { "$ref": "#/definitions/nonEmptyStringArray" }
```

Comment:
Search and replace error, good catch. (although some want this done, I'd rather keep this focused on requirements unless there's immediate agreement to do it for dependencies as well- otherwise I'll fix this to just be for requirements as intended).